doc/configure.md

```diff
@@ -16,7 +16,7 @@ Sōzu configuration process involves 3 major sources of parameters:
 
 Parameters in the global section allow you to define the global settings shared by the master and workers (like the log level):
 
-* `command_socket` path to the unix socket command (see below for more information)
+* `command_socket` path to the unix socket command (see sozuctl for more information)
```

Comment:
again, the command socket is not specific to sozuctl. cf tools like https://github.com/sozu-proxy/sozu-acme for other usage